sig-aws/kubernetes-on-aws.md

```diff
@@ -0,0 +1,19 @@
+Kubernetes on Amazon Web Services
+====
+
+This page lists different options of starting a Kubernetes cluster on Amazon Web Services.
+
+* Clocker: http://www.clocker.io/tutorials/kubernetes-cluster.html
```

Comment:
Can we please alphabetize this?

I know it seems like a burden, but it will greatly reduce the concept of "which way is better".

Also it gives future additions a place to be added instead of "last"

Thoughts?